packages/react-avatar/.eslint.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Check if this is still required, I was getting some error so I needed to do this change

Reply:
removed whole file